modules/shortlinks.php

```diff
@@ -128,5 +128,11 @@ function wpme_rest_get_shortlink( $object ) {
 	add_action( 'rest_api_init', 'wpme_rest_register_shortlinks' );
 }
 
-// Register Gutenberg plugin
-jetpack_register_plugin( 'shortlinks' );
+/**
+ * Set the Shortlink Gutenberg extension as available.
+ */
+function wpme_set_extension_available() {
```

Comment:
lets prefix this with `jetpack_` instead of `wpme`. What is wpme? anyway.

Reply:
It's a prefix that's used for pretty much every other function in that file (while `jetpack_` isn't used there at all yet) -- I think it stands for `wp.me`, the domain that's used for our shortlinks.

I'd rather keep it for consistency with those other functions in that file :grimacing: